server/utils.py

```diff
@@ -1,11 +1,23 @@
+from enum import Enum
 from girder.utility.model_importer import ModelImporter
 from girder import logger
 from girder.models.item import Item
 from girder.models.folder import Folder
 from girder.constants import AccessType
 from girder.api.rest import RestException
 
-from .dataone_register import find_initial_pid
+
+class DataONELocations(Enum):
```

Comment:
Interesting. I'd never seen this pattern but it makes sense. Does the plugin codebase tend to use this pattern for constants or does it tend to use module-level globals?

Reply:
It looks like this could have actually gone in `constants.py` as a `class`. Most of the constants are encapsulated in a `class` rather than subclassing `Enum`.